### DISCUSSION TITLE
On macOS, how can I remove the gap on the left of my prompt when I have both window_margin_width and window_border_width set to 0?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [kovidgoyal/kitty#7387] [Q&A] [answered]
```
window_margin_width 0
window_border_width 0px
```

![CleanShot 2024-04-24 at 06 37 59@2x](https://github.com/kovidgoyal/kitty/assets/42647030/0e376b75-ff66-4782-8e49-c3e39f160ae0)

**Comment by kovidgoyal** [comment, marked as answer]
Make your window size an exact multiple of the cell size or chage the
positioning strategy in kitty.conf to top left

**Reply by MoltenMonster** [reply]
Did you mean `placement_strategy`? `placement_strategy top-left` works, thanks.
